Shared/ViewModels/UserSignInViewModel.swift

```diff
@@ -79,4 +89,60 @@ final class UserSignInViewModel: ViewModel {
         let urlString = ImageAPI.getSplashscreenWithRequestBuilder().URLString
         return URL(string: urlString)
     }
+
+    func startQuickConnect() {
+        QuickConnectAPI.getEnabled()
+            .sink(receiveCompletion: { completion in
+                self.handleAPIRequestError(completion: completion)
+            }, receiveValue: { enabled in
+                self.quickConnectEnabled = enabled
+            })
+            .store(in: &cancellables)
+
+        QuickConnectAPI.initiate()
```

Comment:
Ah okay, the error I was getting on my remote server was because it wasn't enabled and I didn't know we had to check. `initiate()` is still called even when it's disabled, properly getting an error but we shouldn't show _that_ error to the user. So break out this initiate call into a private function `initiateQuickConnect` which is called in `getEnabled` if it's enabled.